app-crypt/nitrocli/nitrocli-0.1.3.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CRATES="
+cc-1.0.25
+hid-0.4.1
+hidapi-sys-0.1.4
+libc-0.2.45
+nitrocli-0.1.3
+pkg-config-0.3.14
+"
+
+inherit cargo
+
+DESCRIPTION="A command line tool for interacting with the Nitrokey Storage."
+HOMEPAGE="https://github.com/d-e-s-o/nitrocli"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+RESTRICT="mirror"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+# We require gnupg for /usr/bin/gpg-connect-agent.
+RDEPEND="
+	app-crypt/gnupg
+	dev-libs/hidapi
+"
+DEPEND="
+	>=dev-lang/rust-1.31.0
+	${RDEPEND}
+"
+
+src_install() {
+	cargo install -j $(makeopts_jobs) --path=. --root="${D}/usr" $(usex debug --debug "") \
+		|| die "cargo install failed"
+	rm -f "${D}/usr/.crates.toml"
```

Comment:
This too.

Reply:
Ok, so if the default cargo_src_install doesnt work, you need to append `|| die` here since it calls for an external command. Also, take the -f out since you should know if this file exists or not. (And if it doesn't then this command is useless anyway)